### PR TITLE
Added Rocket-League project update

### PR DESCRIPTION
**Pull request by hmcty**
### What
News post to provide a small progress report on the current state of the Rocket League project.

### Why
Increase engagement with the outside community and continue building a club history.

## How
Added labeled diagram from integration test meeting and snippets discussing the project team's goals and progress.

### CODE REVIEW

**Comment by jrbaxter0** [comment]
seems the picture is distorted? Is it the same size as the other articles?
image

Otherwise, I'm a big fan. See the two minor comments on wording.

Do you also want to create a post in Projects?

File: _posts/2020-11-02-f20_rocket_league_update.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: article
+categories: news
+title: Rocket League Project Update
+image:
+  feature: news/rl_update_1_feature.png
+  teaser: news/rl_update_1_teaser.png
+author: hmccarty
+---
+Purdue ARC made strides today in one of it's latest projects, Real Life Rocket League. Through the IRL Rocket League project, Purdue ARC hopes to inspire solutions for autonomous systems through a fast-paced, tournament bracket competition. In the future, students from Purdue and outside universities will compete in developing winning strategies for a game of soccer using fully-autonomous RC cars.
```
Comment (jrbaxter0) [inline _posts/2020-11-02-f20_rocket_league_update.md]: Not sure if we should capitalize it like this. The official name is "Rocket League," correct? Can we make this a hyperlink to the project when we make that page?

File: _posts/2020-11-02-f20_rocket_league_update.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Though this goal is still far away, the team is hopeful these ideas will come to fruition. This was shown today as the project team had it's first integration test meeting. Students met up in person, following Protect Purdue safety guidelines, to begin bringing the individual components of the system together. As shown in the photo above, a basic field was setup to demonstrate the project's progress. Currently, the team has a working localization system to track the various field elements and an algorithm to plan complex trajectories. In future integration testing, the team will be finely tuning the RC car's control loops and implementing algorithms to follow the planned paths.
+
+This is one of many steps in achieving the goal set forth by the IRL Rocket League team. By the end of the semester, the team expects to have a fully working RC car. One that is able to plan and follow trajectories to score goals with high accuracy. This will then lead to adding additional cars and increasing the overall functionality of the system.
```
Comment (jrbaxter0) [inline _posts/2020-11-02-f20_rocket_league_update.md]: I'd say "fully working autonomous car" not "RC car." Want to emphasize that we aren't controlling them

**Comment by hmcty** [comment]
> seems the picture is distorted? Is it the same size as the other articles?

I resized it to be the same size as other photos, which is why it's distorted. I'll look later at getting a less distorted version.


> Do you also want to create a post in Projects?

Yes

File: _posts/2020-11-02-f20_rocket_league_update.md (hunk 2 above)
Comment (hmcty) [reply _posts/2020-11-02-f20_rocket_league_update.md]: Agreed

File: _posts/2020-11-02-f20_rocket_league_update.md (hunk 1 above)
Comment (hmcty) [reply _posts/2020-11-02-f20_rocket_league_update.md]: Agreed

**Review by jrbaxter0 — APPROVED** [review]
Would like to see a `project` post in the future, but looking good for now

**Pull request merged**